app/actions/controllers.py

```diff
@@ -99,10 +99,12 @@ def create_action(user_data):
         return
 
     action = Actions(title = user_data["title"])
-    charge.author = user.id
-    charge.description = user_data["description"]
-    charge.charge = charge.id
-    charge.status = ChargeStatusType.InProgress
+    action.author = user.id
+    action.assigned_to = assigned_to.id
+    action.description = user_data["description"]
```

Comment:
You might want to use the get function with a default value. If description isn't in user_data it will throw a KeyError.